notification/ntfnchans.go

```diff
@@ -44,13 +44,13 @@ var NtfnChans struct {
 	UpdateStatusDBHeight              chan uint32
 	SpendTxBlockChan, RecvTxBlockChan chan *txhelpers.BlockWatchedTx
 	RelevantTxMempoolChan             chan *dcrutil.Tx
-	NewTxChan                         chan *mempool.NewTx
+	NewTxChan                         chan *dcrjson.TxRawResult
```

Comment:
We're now forwarding a pointer to the same struct returned by the dcrd notification.